test/cpp/qps/client_async.cc

```diff
@@ -316,23 +317,28 @@ class AsyncClient : public Client {
   }
 
  private:
-  class boolean { // exists only to avoid data-race on vector<bool>
+  class boolean {  // exists only to avoid data-race on vector<bool>
```

Comment:
So, the C++ standard is, IMO, broken on this one. The standard officially says that accesses to disjoint elements of vectors are thread-safe, except for vector<bool>. Compilers optimize are allowed to optimize that and can let it be unsafe.